### PR TITLE
Add GOOGLE_PLAY_CONSOLE and APP_STORE_CONNECT sources

### PR DESCRIPTION
**Pull request by lspecian**
Add GOOGLE_PLAYCONSOLE and APPLE_APPSTORECONNECT sources

### CODE REVIEW

**Comment by pfrisella** [comment]
It looks like these sources should just be Google Play by Google and App Store Connect by Apple. The goal is to register the underlying data source, not connector specific versions. For example, if there are multiple connectors that query Twitter data, they all use the same TWITTER source, there isn't an entry for each connector.

**Comment by lspecian** [comment]
hello @pfrisella , I initially misunderstood the purpose of the files, I believe that the sources are now changed to reflect the proper information.

File: sources/apple_appstoreconnect.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "APPLE_APPSTORECONNECT",
```
Comment (pfrisella) [inline sources/apple_appstoreconnect.json:2]:
```suggestion
  "id": "APP_STORE_CONNECT",
```

Comment (pfrisella) [inline sources/apple_appstoreconnect.json:2]: if the ID is changed as suggested then the file name should be app_store_connect.json

File: sources/google_playconsole.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "GOOGLE_PLAYCONSOLE",
```
Comment (pfrisella) [inline sources/google_playconsole.json:2]:
```suggestion
  "id": "GOOGLE_PLAY_CONSOLE",
```

File: sources/google_playconsole.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (pfrisella) [inline sources/google_playconsole.json:1]: if the ID is changed as suggested then the file name should be google_play_console.json

**Review by pfrisella — CHANGES_REQUESTED** [review]

**Review by pfrisella — APPROVED** [review]
LGTM. Thanks for adding!

**Pull request merged**